llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -612,6 +612,118 @@ Instruction *emitSpecConstantRecursive(Type *Ty, Instruction *InsertBefore,
 
 } // namespace
 
+/// Function creates load instruction from the given Buffer by the given Offset.
+/// Function returns the value of load instruction.
+static Value *createLoadFromBuffer(CallInst *CI, Value *Buffer, size_t Offset,
```

Comment:
It seems like we are using anonymous namespace approach to hide internal functions in this file, I suggest to conitnue with it to have unified codebase. You just need to move `} // namespace` on line 613 a bit